.github/workflows/jdk-17-tests.yml

```diff
@@ -19,13 +19,13 @@ jobs:
       - name: Build in jdk11
         run: mvn -B clean install -DskipTests --file pom.xml
       - name: Set up JDK 17
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
-          distribution: 'adopt'
+          distribution: 'temurin'
           java-version: 17
       - name: check mvn version
         run: echo "MAVEN_VERSION=$(mvn -v)"
       - name: check java version
         run: echo "JAVA_VERSION=$(java -version)"
       - name: JDK 17 Tests
-        run: mvn -B clean install --file integration/pom.xml -fae
```

Comment:
Let's keep -fae flag, so that the build will continue with non-impacted modules even-though a test failure occurs.